static/src/stylesheets/inline/story-package-garnett.scss

```diff
@@ -128,6 +127,10 @@ $story-package-dynamic-card-text: $story-package-card-colour;
         }
     }
 
+    .fc-item__title {
+        font-weight: 800;
```

Comment:
Maybe one for the future but is it possible to pull this out and give it a (sass variable) name? Not sure if font weight of 800 is something we use elsewhere.